src/Sarif.UnitTests/Writers/SarifLoggerTests.cs

```diff
@@ -52,7 +59,13 @@ public void SarifLogger_RedactedCommandLine()
                 string pathToExe = Path.GetDirectoryName(Assembly.GetCallingAssembly().Location);
                 string commandLine = Environment.CommandLine;
 
-                if (pathToExe.IndexOf(@"\Extensions", StringComparison.OrdinalIgnoreCase) != -1)
+                if (commandLine.Contains("testhost.dll") || commandLine.Contains("\\xunit.console"))
```

Comment:
>testhost.dll [](start = 42, length = 12)

is it ok that this isn't case-sensitive?

Reply:
I don't think Environment.CommandLine would return inconsistent values.  Since this is a test case, it's probably better to be safe than sorry though.

---
In reply to: [141766792](https://github.com/Microsoft/sarif-sdk/pull/624#discussion_r141766792) [](ancestors = 141766792)